### DISCUSSION TITLE
How to enable Ingress with TLS in Skywalking

### DISCUSSION THREAD

**Discussion by Jatin3708** [apache/skywalking#12869] [Q&A]
Hi,

I have enabled ui.ingress.enabled as true but my deployment is failing with error : 

`│ Error: failed to create resource: Ingress.networking.k8s.io "skywalking-skywalking-helm-ui" is invalid: spec: Invalid value: []networking.IngressRule(nil): either `defaultBackend` or `rules` must be specified`

While as per values.yaml file those are already passed, so anything else need to do over here ?

Also how to enabled TLS in helm chart ?

**Comment by wu-sheng** [comment]
We don't add those codes, although it should be easy to add(web UI is a Spring Web container).

So, generally, you could either write some codes to add that, or add an extra Gateway(Envoy Gateway, Nginx) etc. to manage TLS relative things.
